src/lib/containers/widgets/query-filter/QueryFilter.tsx

```diff
@@ -111,15 +111,15 @@ export function applyChangesToValuesColumn(
 export const applyMultipleChangesToValuesColumn = (
   lastRequest: QueryBundleRequest | undefined,
   facet: FacetColumnResultValues,
-  onChangeFn: Function,
-  facetNameMap?: {},
+  onChangeFn: (result: FacetColumnRequest[]) => void,
+  facetNameMap?: Record<string, string>,
 ) => {
   const facetNames = (facetNameMap && Object.keys(facetNameMap)) || []
   if (facetNames.length) {
     facet.facetValues.forEach(facetValue => {
       if (facetNames.includes(facetValue.value)) {
         facetValue.isSelected = facetNameMap
-          ? facetNameMap[facetValue.value]
+          ? !!facetNameMap[facetValue.value]
```

Comment:
This is technically a functional change but I think this was the original intent